draft-hendrickson-privacypass-public-metadata-issuance.md

```diff
@@ -43,10 +46,220 @@ TODO Abstract
 
 TODO Introduction
 
+# Terminology
+
+{::boilerplate bcp14}
+
+The following terms are used throughout this document.
+
+- Public Metadata: Arbitrary length metadata that can be viewed by the client, attester, issuer, and verifier. After signing metadata is bound to the token & signature and cannot be mutated as described in TODO_link_crypto_draft. In literature this may also be referred to as 'partial blinding'. We will refer to it as public metadata throughout this draft and privacy pass.
+
+# Issuance Protocol for Publicly Verifiable Tokens {#public-flow}
+
+This section describes a variant of the issuance protocol in {{Section 6 of PROTOCOL}}
+for producing publicly verifiable tokens including public metadata using {{PBLINDRSA}}.
+In particular, this variant of the issuance protocol works for the
+TODO_insert_variants of the blind RSA protocol variants described in {{Section 5 of BLINDRSA}}.
+
+The public metadata issuance protocol differs from the protocol in
+{{Section 6 of PROTOCOL}} in that the tokens (both before and after signing) contain metadata set by the client that is visible to the signer (issuer) and verifier. This means any Client can set metadata on a token. Any issuer or verifier may reject the token by aborting the protocol, but cannot set or mutate the metadata.
+
+Beyond this difference, the publicly metadata issuance protocol variant is
+nearly identical to the publicy verifiable issuance protocol variant. In
+particular, Issuers provide a Private and Public Key, denoted skI and pkI,
+respectively, used to produce tokens as input to the protocol. See
+{{Section 6.5 of PROTOCOL}} for how this key pair is generated.
```

Comment:
I think we may want to include key generation details in this document, in particular because the PBRSA variant requires a very specific type of key.

Reply:
More generally wondering if we should avoid stacking references to other token types, and just redeclare type specific algorithm/structures in this document as well?

Reply:
Replaced with a TODO that will review token generation, once that draft is ready.